debian/snapd.install

```diff
@@ -2,3 +2,4 @@ usr/bin/snap
 usr/bin/snap-exec /usr/lib/snapd/
 usr/bin/snapctl
 usr/bin/snapd /usr/lib/snapd/
+usr/lib/snapd/system-shutdown
```

Comment:
mvo confirmed that thee is no 'part that gets intalled i core images', and that creating another package would be a pain for sru'ing and such